package.json

```diff
@@ -64,5 +64,8 @@
   "overrides": {
     "minimatch": "^9.0.3",
     "nth-check": "^2.1.1"
+  },
+  "dependencies": {
+    "playwright": "^1.38.1"
```

Comment:
Thanks for your great job! 
But could you please tell me why need to install `playwright` here? If we do need to install it, why not put it into `ketcher-autotests/package.json` and install it as `devDependencies`?